apollo-federation/src/sources/connect/validation/graphql.rs

```diff
@@ -0,0 +1,25 @@
+//! Helper structs & functions for dealing with GraphQL schemas
+use std::ops::Deref;
+
+use apollo_compiler::Name;
+use apollo_compiler::Schema;
+use line_col::LineColLookup;
+
+mod strings;
+
+pub(super) use strings::GraphQLString;
+
+pub(super) struct SchemaInfo<'schema> {
+    pub(crate) schema: &'schema Schema,
+    pub(crate) lookup: LineColLookup<'schema>,
```

Comment:
I'm wary of including dependencies in public API (even `pub(crate)`) as it makes it more difficult to remove or update the dependency. Especially in this case, where there is really only a single method on the dependency type, this would be very easy to just wrap with a method on this struct rather than exposing the `LineColLookup` directly. Then pass around `SchemaInfo` instead of the `lookup` field.

Reply:
So you're suggesting a `SchemaInfo::get_line_col` which calls, internally to `lookup.get`?;

Reply:
yes.